### ISSUE TITLE
The arrows "<" and ">" should be hidden by screen readers. This ensures

### ISSUE THREAD

**Issue by TheOneRing** [owncloud/client#11635]

**Comment by erikjv** [comment]
I think we should remove the arrows completely

**Comment by TheOneRing** [comment]
Can we have them be managed by the qpa?

**Comment by erikjv** [comment]
Apparently we're not using a `QWizard` here, so I changed our button logic to be the same as what `QWizard` does.
